.github/workflows/ci_summary.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI Summary
+on:
+  schedule:
+    # run daily 8:00 CET on master branch
+    - cron: '0 7 * * *'
+jobs:
+  build_summary:
```

Comment:
If I am not mistaken, this will run on every fork of this repo, including our private one.

We've seen issues with this previously, where we have a lot of runs on the private repo that eats into our credits. We also don't want other forks to run this so should add some type of conditional:

https://github.community/t/do-not-run-cron-workflows-in-forks/17636

Reply:
We have cron jobs with the same setup, which runs every night and it doesn't run on forks. According to the [documentation](https://docs.github.com/en/actions/reference/events-that-trigger-workflows):
>Scheduled workflows run on the latest commit on the default or base branch.

So I don't see an issue.